src/main/java/spotify/api/enums/AuthorizationScope.java

```diff
@@ -1,5 +1,16 @@
 package spotify.api.enums;
 
+/**
+ * This class contains and sets the suitable values for the authorization scope. Scopes provide Spotify 
```

Comment:
I would replace `This class contains and sets the suitable values for the authorization scope` with `This class contains all possible scope values that the user can authorize for.` The rest of the Javadoc is fine.